EIPS/eip-5851.md

```diff
@@ -0,0 +1,257 @@
+---
+eip: 5851
+title: On-chain verifiable claims.
+description: Interface for contracts that manage verifiable claims and identifiers as Soulbound tokens.
+author: Yu Liu (@yuliu-debond), Junyi Zhong (@Jooeys)
+discussions-to: https://ethereum-magicians.org/t/eip-5815-kyc-certification-issuer-and-verifier-standard/11513
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-10-18
+requires: 721, 1155, 1167, 1967
+---
+## Abstract
+
+This interface provides a means of issuing and verifying on-chain certifications, specifying the claims as on-chain metadata.
```

Comment:
[EIP-1](https://eips.ethereum.org/EIPS/eip-1#what-belongs-in-a-successful-eip) recommends that the Abstract be a multi-sentence paragraph containing a technical summary.

While this paragraph meets the terseness requirement, I don't think it gives enough of an overview of the proposal. I'd like to see a bit more content here. Maybe something like:

```suggestion
This proposal introduces a method of certifying that a particular address meets a claim, and a method of verifying those certifications using on-chain metadata. Claims are conditions that may be satisfied (for example: `age >= 18`), and are certified by issuers using a Soulbound Token (SBT).
```

Reply:
@yuliu-debond, I have added this change but if you can recommend an abstract that explains comprehensively.

Reply:
I rewrite it as "Claims are assertions or statements made about a subject having certain properties that may be met conditions (for example: `age >= 18`), and are certified by issuers using a Soundbound Token(SBT)."